pkg/api/handler/default.go

```diff
@@ -187,13 +186,16 @@ func (m ArcDefaultHandler) GETTransactionStatus(ctx echo.Context, id string) err
 		return ctx.JSON(e.Status, e)
 	}
 
+	emptyString := ""
+
 	return ctx.JSON(http.StatusOK, api.TransactionStatus{
 		BlockHash:   &tx.BlockHash,
 		BlockHeight: &tx.BlockHeight,
 		TxStatus:    &tx.Status,
 		Timestamp:   m.now(),
 		Txid:        tx.TxID,
 		MerklePath:  &tx.MerklePath,
+		ExtraInfo:   &emptyString,
```

Comment:
You could use this instead of creating that empty string variable `PtrTo(""),`

Reply:
same here, `PtrTo` is undefined.

Reply:
Have you checked? 
It is defined

```
	return ctx.JSON(http.StatusOK, api.TransactionStatus{
		BlockHash:   &tx.BlockHash,
		BlockHeight: &tx.BlockHeight,
		TxStatus:    &tx.Status,
		Timestamp:   m.now(),
		Txid:        tx.TxID,
		MerklePath:  &tx.MerklePath,
		ExtraInfo:   PtrTo(""),
	})
```

Reply:
Oh, it should work here